src/main/kotlin/ch/hsr/dsl/dwrtc/signaling/ClientService.kt

```diff
@@ -222,22 +222,22 @@ class ClientService constructor(peerPort: Int? = findFreePort()) : IClientServic
         fun dispatchMessage(clientMessage: ClientMessage, senderPeerAddress: PeerAddress) {
             val recipientSessionId = clientMessage.recipientSessionId!!
             val senderSessionId = clientMessage.senderSessionId!!
+
             emitterMap[recipientSessionId]?.let {
                 logger.info { "message accepted, found emitter for $recipientSessionId" }
                 it(ExternalClient(senderSessionId, senderPeerAddress, peer), clientMessage)
-            } ?: run {
-                logger.info { "message discarded (no registered emitter for session id $recipientSessionId" }
-            }
+            } ?: logger.info { "message discarded (no registered emitter for session id $recipientSessionId" }
         }
 
         /** Only dispatch a message if it's actually one of our own messages */
-        fun tryDispatchingMessage(messageDto: Any?, senderPeerAddress: PeerAddress): Any {
+        fun tryDispatchingMessage(messageDto: Any?, senderPeerAddress: PeerAddress): Boolean? {
```

Comment:
```suggestion
        fun tryDispatchingMessage(messageDto: Any?, senderPeerAddress: PeerAddress): Boolean {
```

Not sure, but does it need to be nullable? I just added the suggestion so it's easier to merge, if it can be non-nullable.